torch/nested/_internal/ops.py

```diff
@@ -502,16 +509,80 @@ def zero__default(func, *args, **kwargs):
 
 
 @register_jagged_func(
-    torch.ops.aten._softmax.default, "self: jt, dim: any, half_to_float: any"
+    torch.ops.aten._softmax.default, "self: jt_all, dim: any?, half_to_float: any"
```

Comment:
nit: why is `dim` changed to `any?`? is dim optional?

Reply:
that's a good point; i was getting some errors with `dim` being non-optional for `sum` and `mean` so wanted to prevent that for `softmax`, but it should definitely require a `dim`!